wp-content/themes/twentyseventeen-wp20/style.css

```
@@ -943,6 +943,13 @@ body:not(.title-tagline-hidden) .site-branding-text {
 	color: var(--wp20--color--primary);
 }
 
+/*
+ * All content
+ */
+.entry-content img {
```

Could exclude the swag images here, rather than turning off again on line 1527
```suggestion
.entry-content img:not(.downloads-item-preview img) {
```

---

That WFM 👍🏻